org.uqbar.project.wollok/src/org/uqbar/project/wollok/interpreter/WollokInterpreterEvaluator.xtend

```diff
@@ -284,16 +285,20 @@ class WollokInterpreterEvaluator implements XInterpreterEvaluator {
 	}
 	
 	def createNamedObject(WNamedObject namedObject, String qualifiedName) {
-		namedObject.hookObjectInHierarhcy(classFinder)
+		namedObject.hookObjectInHierarchy(classFinder)
```

Comment:
No, and yes.
Just that dev-1.4 and mixins branches were not updated yet.
Once I merge mixins into dev-1.4 I will merge dev -> dev1.4 and will be removed